app/models/profile.rb

```diff
@@ -333,4 +335,17 @@ def send_push_notifications
     event = PushNotification::ReproofCompletedEvent.new(user: user)
     PushNotification::HttpPush.deliver(event)
   end
+
+  def biometric?(level)
+    ::User::BIOMETRIC_COMPARISON_IDV_LEVELS.include?(level)
+  end
+
+  def track_biometric_reproof
+    SpUpgradedBiometricProfile.create(
+      user_id: user_id,
```

Comment:
WDYT about defining a user association on the model so we could just pass in the user? it would still get written as a user ID directly
```suggestion
      user: user,
```